app/controllers/questions_controller.rb

```diff
@@ -55,6 +55,10 @@ def search
     @questions = Question.search(params[:q])
   end
 
+  def by_tags
+    @questions = Question.by_tags(params[:tags_id])
```

Comment:
Cool. I see what you talking about - the index jbuilder view. Makes sense. Thanks.